tests/compose-rootfs/Containerfile

```diff
@@ -17,12 +17,6 @@ COPY . /src
 WORKDIR /src
 RUN --mount=type=bind,from=repos,src=/,dst=/repos <<EORUN
 set -xeuo pipefail
-# Synchronize the dnf/rpm configs from the repos container.
-for x in etc/dnf etc/yum.repos.d etc/pki/rpm-gpg; do
```

Comment:
Yeah here's a maddening thing, `--source-root` is basically still nerfed because of `gpgkeypath=` in the default repo files; what libdnf *should* be doing here is prefixing those with the source root.

So for now in reality when doing cross builds we still need at least
`cp -a /repos/etc/pki/rpm-gpg/* /etc/pki/rpm-gpg`
but that's *still* not the only problem! We have dnf variable issues too...

I'll try to fix all this and we can change this example to be a real cross build, but for now it at least has basic coverage of not having `.repo` files in the manifest dir.